qos-monitor/documentation/QualityOfServiceMonitor-IDD.md

```diff
@@ -115,8 +115,8 @@ Returns HTTP-OK in order to confirm received event notification.
 | Field | Description | Necessity | Format/Limitations |
 | ----- | ----------- | --------- | ----------- |
 | `eventType` | Type of event. | mandatory | must be "INTERRUPTED_MONITORING_MEASUREMENT" |
-| `metaData` |  The "key - value" pairs for event filtering. | mandatory | max.length = 65535, must contain a "processID" key assosiated with a string value, which is parsable to UUID object, must contain an "exception" key assosiated with a string value,  may contain an "root_cause" key assosiated with a string value|
-| `payload` | String representation of the event. | mandatory | must be an emty list as "{[]}" |
+| `metaData` |  The "key - value" pairs for event filtering. | mandatory | max.length = 65535, must contain a "processID" key associated with a string value, which is parsable to UUID object, must contain an "exception" key associated with a string value,  may contain an "root_cause" key associated with a string value|
```

Comment:
According to the QoSMonitorConstants, the keys must be EXCEPTION and ROOT_CAUSE, not the lower-case version

Reply:
done